presto-main/src/main/java/io/prestosql/sql/planner/plan/WindowNode.java

```diff
@@ -334,12 +334,12 @@ public Function(
                 @JsonProperty("signature") Signature signature,
                 @JsonProperty("arguments") List<Expression> arguments,
                 @JsonProperty("frame") Frame frame,
-                @JsonProperty("ignoreNulls") Boolean ignoreNulls)
+                @JsonProperty("isIgnoreNulls") boolean ignoreNulls)
```

Comment:
This needs to be `@JsonProperty("ignoreNulls")` since `ignoreNulls` is what the name will be in the JSON (the JSON property are the logical Java Bean names -- without the `is` / `get` prefixes and starting with lowercase)